roles/openshift_node/tasks/main.yml

```diff
@@ -41,6 +41,9 @@
   notify:
   - restart openshift-node
 
+- name: Allow NFS access for VMs
+  seboolean: name=virt_use_nfs state=yes persistent=yes
```

Comment:
@jwhonce is this something that we want to just enable without making a flag for it?  I'm good with it being the default, but I think in the long run we'd want it to be configurable along with the other persistent volume options (ideally triggered by the associated configuration of the persistent storage as well).